src/views/CountDetail.vue

```diff
@@ -555,10 +649,39 @@ async function readyForReview() {
   });
   await alert.present();
 }
+
+async function findProduct(sku) {
+  if(!sku) {
+    showToast(translate("Scan a valid product sku"));
```

Comment:
When sku is not found we are displaying a toast here, but as we are returning an empty value after this, then another toast will be displayed from line https://github.com/hotwax/inventory-count/pull/434/files#diff-02a864e08f57bf16805f1b7d39a144cc63906029bcaaa582135dd2ceeeaa7364R380. We should handle this case so that a single toast message is displayed.